docs/release-notes/NuGet-6.1.md

```diff
@@ -0,0 +1,121 @@
+---
+title: NuGet 6.1 Release Notes
+description: Release notes for NuGet 6.1 including new features, bug fixes, and DCRs.
+author: zivkan
+ms.author: zivkan
+ms.date: 2/15/2022
+ms.topic: conceptual
+---
+
+# NuGet 6.1 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.1.0**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.1](https://visualstudio.microsoft.com/downloads/) | [6.0.200](https://dotnet.microsoft.com/download/dotnet-core/6.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with.NET Core workload
+
+## Summary: What's New in 6.1
+
+* Show help only for the error itself has to do with a command, argument, or option being invalid with `dotnet nuget` command - [#10788](https://github.com/NuGet/Home/issues/10788)
```

Comment:
Could we please update the issue title and the release notes so that we show what's the new behavior in the release rather than the problem? For example, 10788 would be better described as: "show subcommand help instead of main dotnet nuget trust command help for all cases." 

We are already doing the above for 11091, so we only need to address this one.

Reply:
@aortiz-msft - I addressed your comment. Please review whenever you have a minute so that I can merge this PR ASAP.